lib/rucio/core/replica.py

```diff
@@ -799,6 +808,48 @@ def _list_replicas_for_files(file_clause, state_clause, files, rse_clause, updat
             {'scope': scope, 'name': name} in files and files.remove({'scope': scope, 'name': name})
 
 
+def get_vp_endpoint():
+    vp_endpoint = config_get('virtual_placement', 'vp_endpoint', default='')
+    print('vp endpoint: ', vp_endpoint)
+    return vp_endpoint
+
+
+def get_multi_cache_prefix(cache_site, filename, logger=logging.log):
+
+    vp_endpoint = get_vp_endpoint()
+    if not vp_endpoint:
+        return ''
+
+    # print('Looking up prefix for cache:', cache_site, 'and filename:', filename)
+    x_caches = REGION.get('CacheSites')
+    if x_caches is NO_VALUE:
+        logger(logging.DEBUG, 'reloading xcache sites and ranges.')
+        try:
+            response = requests.get('{}/serverRanges'.format(vp_endpoint), verify=False)
+            if response.status_code == 200:
+                x_caches = response.json()
+                logger(logging.DEBUG, x_caches)
```

Comment:
this would just dump some object to the log and when a server has load this will not directly be after `reloading xcache sites and ranges.`. This does not seem to be useful.

Reply:
OK. removed